.github/workflows/trivy.yml

```diff
@@ -0,0 +1,43 @@
+name: Container Security
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '29 19 * * 4'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    permissions:
+      contents: read 
+      security-events: write 
+      actions: read
+    name: Build
+    runs-on: "ubuntu-20.04"
```

Comment:
that's quite old version, why not `ubuntu-latest`?

Reply:
@derrix060 is indeed quite old, but this version is mentioned in all their action examples and is commonly used for this action, at least as far as I know. However, we can certainly try `ubuntu-latest`